include/deal.II/base/exceptions.h

```diff
@@ -180,6 +180,17 @@ class ExceptionBase : public std::exception
 /**
  * Declare an exception class derived from ExceptionBase without parameters.
  *
+ * @note This and similar macro names are examples of preprocessor definitions
+ * in the deal.II library that are not prefixed by a string that likely makes
+ * them unique to deal.II. As a consequence, it is possible that other
+ * libraries your code interfaces with defines similar names, and the result
```

Comment:
defines -> define
similar names -> the same name

Reply:
fixed